### PR TITLE
Community model updates

### PR DESCRIPTION
**Pull request by terichadbourne**
Updates to reflect the new ProtoSchool community model. Drafting and collecting feedback here  for a cleaner announcement process, but this will ultimately be merged into https://github.com/protoschool/organizing. 

This PR will be merged AFTER https://github.com/ProtoSchool/protoschool.github.io/pull/365, which adds the events page to the website. 

After both PRs are merged, I'll update current chapter leaders and those who've requested chapters, referring to the docs here as a reference point.  

Key changes include: 
- Reworking README.md to reflect the new model (intended for all readers, including new leaders)
- Adding COMMUNITY_MODEL_UPDATE.md as a resource for current chapter leaders making the transition
- Removing the chapter request issue template that's no longer needed

### CODE REVIEW

File: COMMUNITY_MODEL_UPDATE.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  "city": "San Francisco, CA",
+  "country": "US",
+  "region": "North America",
+  "organizers": ["mikeal"],
+  "website": "https://www.meetup.com/dweb-san-francisco/",
+  "repo": "http://github.com/protoschool/san-francisco",
+  "twitter": "sfprotoschool"
+}
+```
+After April, the chapters page will be removed from our website and learners will discover your group exclusively through our [events page](https://proto.school/#/chapters). (We'll provide a redirect for any visitors looking for the old chapters page.)
```
Comment (zebateira) [inline COMMUNITY_MODEL_UPDATE.md]:
```suggestion
After April, the chapters page will be removed from our website and learners will discover your group exclusively through our [events page](https://proto.school/#/events). (We'll provide a redirect for any visitors looking for the old chapters page.)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,56 +25,25 @@ We also take a _lot_ of inspiration from [NodeSchool](https://nodeschool.io/). (
...
-
-Have an idea for a new tutorial?
+We welcome new contributors to the ProtoSchool community and there are many ways to help, regardless of your technical background. Check out our website for the [many ways to contribute](https://proto.school/#/contribute).
 
-Take a look at the [tutorial ideas](https://github.com/ProtoSchool/protoschool.github.io/labels/new-tutorial) flagged with the `new-tutorial` tag in the ProtoSchool issue queue. If there's a similar idea there already, join the conversation!
+You'll find relevant project info split across a variety of repos:
 
-If you have an idea for a new tutorial that's not yet been proposed, please [open an new issue](https://github.com/ProtoSchool/protoschool.github.io/issues/new?&labels=new-tutorial&template=tutorial-proposal.md&title=New+Tutorial%3A+%5BProposed+title%5D) so we can share feedback before you get started. (Don't feel like you have the skills to build a tutorial yourself? Please still go ahead and share your idea.)
+-   [`organizing` repo](https://github.com/protoschool/protoschool.github.io): local event leadership resources
```
Comment (zebateira) [inline CONTRIBUTING.md]:
```suggestion
-   [`organizing` repo](https://github.com/protoschool/organizing): local event leadership resources
```

File: README.md [hunk 3]
```diff
@@ -1,96 +1,88 @@
...
+# Organizing Local ProtoSchool Events
 
-Organizing a ProtoSchool chapter? Wish you were? You're in the right place!
+If you're looking for information on hosting ProtoSchool workshops in your community, you're in the right place! Read on to learn more about ProtoSchool's community event model and how you can contribute. It's easier than ever to get started!
 
-_For more background on ProtoSchool or other ways to get involved, like building tutorials, check out our [guide to contributing](./CONTRIBUTING.md) or view our [launch presentation](https://github.com/protoschool/organizing/blob/master/presentations/ProtoSchool_Launch_Preso_2019-01-14.pdf)._
+**Quick links for local event leaders**:
+- [View upcoming events](https://proto.school/#/events)
+- [Add your event to our list](https://forms.gle/t1iEzpQAFSYHhpBr9)
+- [Explore our event resources](,/RESOURCES.md)
```
Comment (zebateira) [inline README.md]:
```suggestion
- [Explore our event resources](./RESOURCES.md)
```

**Review by zebateira — CHANGES_REQUESTED** [review]
Looking good 👍 
Just 3 typos.

**Review by olizilla — APPROVED** [review]
Clearly explains what is happening and why. It's really good!

**Review by zebateira — APPROVED** [review]

**Pull request merged**